lib/builderator/control/version.rb

```diff
@@ -32,6 +32,16 @@ class Version
       class << self
         def current
           @current ||= SCM.tags.last
+
+          if @current.nil? && Util.relative_path('VERSION').exist?
+            @current = Version.from_string(Util.relative_path('VERSION').read)
+          end
+
+          if @current.nil?
+            fail 'No current version found! Create a VERSION file or set a version tag in your SCM.'
```

Comment:
@fmitchell-r7 Don't we normally set RuboCop to use `raise` over `fail` (as much as I prefer `fail` myself)? 😉